src/devices/Neo-M8/NeoM8.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO.Ports;
+using Iot.Device.Nmea0183;
+using Iot.Device.Nmea0183.Sentences;
+
+namespace Iot.Device.Gps
+{
+    public sealed class NeoM8 : IDisposable
+    {
+        private SerialPort _sp;
+
+        /// <summary>
+        /// Constructs object representing Neo-M8
+        /// </summary>
+        /// <param name="serialPortName">Name of the serial port to use for communication</param>
+        public NeoM8(string serialPortName)
+        {
+            _sp = new SerialPort(serialPortName);
+            _sp.NewLine = "\r\n";
+            _sp.Open();
+
+            // Device streams continuously and therefore most of the time we would end up in the middle of the line
+            // therefore ignore first line so that we align correctly
+            _sp.ReadLine();
```

Comment:
Should we have an optional bool in the constructor in order to not do a ReadLine here in case we are not in the middle of the line and we don't want to ignore? Or do you think that won't be useful?

Reply:
Currently I don't think it will be useful since you'd have to connect some kind of relay to the NEO-M8 in order to shut it down and otherwise it won't stop talking. We can add it if we find it's useful one day. Also the device repeats measurements every second so there is no loss